msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md

```diff
@@ -140,6 +140,35 @@ In the **Publish** section, you can publish your app to your org or to the Teams
 
 * **App package**: The app package describes how your app is configured that includes app features, required resources, and other important attributes in manifest. The app definition refers to the configuration and settings associated with the application. The Icon tab shows the icon used for your app.
 
+* **App validation**: The app validation tool evaluates your app package based on the test cases that Microsoft uses to review your app. The app validation tool identifies any errors or warnings in your app package and provides guidelines for resolution.
+
+    If your app failed any test cases, you can review the guidelines and make necessary updates to your app package based on the Microsoft test case results. You can also download your app package and validation report.
+
+    <br>
+    <details>
+    <summary><b>Validate your app package</b></summary><br>
+    To ensure that your app functions as expected on Teams, use the app validation tool to check your app package against Microsoft's test cases:
+
+    1. In the Developer Portal, select the **Apps** from the left pane and select the app that you're working.
+    1. Go to **Publish** > **App validation** > **Get started**.
+
+        :::image type="content" source="../../assets/images/tdp/app-validation-home.png" alt-text="Screenshot shows you the app validation in Teams developer portal.":::
+
+    1. Select the validation acknowledgments checkbox and select the **Start validation**.
+
+        :::image type="content" source="../../assets/images/tdp/app-validation.png" alt-text="Screenshot shows you the app validation acknowledgment of your app package.":::
+
+        Teams Developer Portal now starts to validate your app package:
+
+        :::image type="content" source="../../assets/images/tdp/app-validation-in-progress.png" alt-text="Screenshot shows you the app validation in progress.":::
+
+    1. After validation, it shows the list of test cases your app has passed or failed.
```

Comment:
Provide more information about various validation result types.
1. Success - Test case is successful.
2. Warning - Warnings - good to have.
3. Error - Developer must take an action to fix these test cases, else app will not be published.
4. skipped - Unable to execute the test case. Retry by submitting new validation request.

Reply:
Updated as suggested